src/debugpy/server/cli.py

```diff
@@ -41,13 +43,14 @@
 
 class Options(object):
     mode = None
-    address = None
+    address: "tuple[str, int] | None" = None
```

Comment:
I think the quotes are unnecessary?
```suggestion
    address: tuple[str, int] | None = None
```

Reply:
They're only unnecessary on newer Python versions. Debugpy has to run on 3.8 I believe.

![image](https://github.com/microsoft/debugpy/assets/19672699/6a9b0e47-6e24-40ce-8f98-5a79b14d00ef)

Reply:
Oh, interesting, I haven't run into that before.

Reply:
Yeah debugpy's pyproject forces 3.8:

```
[tool.pyright]
pythonVersion = "3.8"
```